internal/cli/iam/organizations/describe.go

```diff
@@ -27,11 +27,17 @@ import (
 	"github.com/spf13/cobra"
 )
 
-const describeTemplate = `ID	NAME
+const (
+	describeTemplateCloud = `ID	NAME
```

Comment:
I see this change and also  new file, which one is it?

Reply:
New files is the approach we'll take.
The above will be removed as part of https://jira.mongodb.org/browse/CLOUDP-168386

Reply:
Why not remove it here if unused?

Reply:
Assuming that then this file should remain unchanged?

Reply:
Also are we certain e2e run against the new added commands?

Reply:
> Why not remove it here if unused?
> Assuming that then this file should remain unchanged?

Organizations was migrated with `opts.InitConditionalOutput`, rather than creating new commands
Organization Invitations was migrated with new commands

I don't want to change it in this PR because it would mean introducing a lot more changes, which I don't want to do while the branch is stable and we're looking to merge back to master (but will do in the follow-up PR)


> Also are we certain e2e run against the new added commands?


Yep. The commands are part of the atlas binary (see [reference change](https://github.com/mongodb/mongodb-atlas-cli/pull/1820/files#r1157286127)), the e2e tests for the new commands still run and pass with the updated references.